scripts/write-presubmit-scheduler.js

```diff
@@ -0,0 +1,83 @@
+#!/usr/bin/env node
+
+const path = require('path');
+const fs = require('fs');
+const GitHubApi = require('github');
+const github = new GitHubApi();
+
+
+/** CONFIGURATION: change these things if you want to tweak how the runs are made. */
+
+/** Path to the local ng-material. By default based on the location of this script. */
+const localRepo = path.resolve(__dirname, '..',);
```

Comment:
The second command seems to be unnecessary.

Reply:
Done